nixos/modules/services/web-servers/nginx/vhost-options.nix

```diff
@@ -31,12 +31,12 @@ with lib;
         options = {
           addr = mkOption {
             type = str;
-            description = lib.mdDoc "IP address.";
+            description = lib.mdDoc "Listen address.";
           };
           port = mkOption {
-            type = port;
+            type = types.nullOr port;
             description = lib.mdDoc "Port number.";
-            default = 80;
+            default = null;
```

Comment:
```suggestion
            default = 80;
```

Reply:
I have no strong opinion on this one, it's true that the default is more readable this way, or the description should reflect that the default is 80 because of NGINX. As you wish @dongcarl.

Reply:
I might be mistaken, but if the default is `80`, then wouldn't unix socket users need to explicitly specify this?
```
{
  addr = "unix:/var/run/nginx.sock";
  port = null;
}
```

I'm happy to add on to the description if people feel strongly about it, but not sure that we should in case `nginx`'s behavior for the default changes in the future.

Reply:
Hm... I think we can go with the current state and see if we need to adjust this later on.

Reply:
This should have been at least mentioned in the description. I expect most people to be a bit confused about this now.